### PR TITLE
Timer Bug, Mobile compatibility and social sites icons adjustment changes introduced in site

### PR DESCRIPTION
**Pull request by bordia98**
- Timer Bug
-  Mobile compatible view for buttons and the main logo
- social icons adjusted

### CODE REVIEW

**Comment by pvgupta24** [comment]
@bordia98 , please use better PR titles that indicate the changes.

File: js/main.js [hunk 1]
```diff
@@ -243,12 +243,12 @@ $(document).ready(function(){
         var seconds = Math.floor((distance % (1000 * 60)) / 1000);
```
Comment (pvgupta24) [inline js/main.js]: Update the timer to show doubly digit figures by :
`var formattedNumber = ("0" + myNumber).slice(-2);`

File: js/main.js [hunk 2]
```diff
@@ -262,8 +262,8 @@ $(document).ready(function(){
             var seconds = Math.floor((distance % (1000 * 60)) / 1000);
             
             // Output the result in an element with id="demo"
-            document.getElementById("timer").innerHTML ="<div class='start-in'>ends in:</div>"+ days + "<span>days  </span>: " + hours + "<span>hour</span>: "
-            + minutes + "<span>mins  </span>: " + seconds + "<span>secs  </span>";
+            document.getElementById("timer").innerHTML = days + "<span>days</span>: " + hrs + "<span>hrs</span>: "
+            + minutes + "<span>m</span>: " + seconds + "<span>s</span>";
```
Comment (pvgupta24) [inline js/main.js]: Can you decrease the spacing between days, hours, minutes and seconds?

File: js/main.js (hunk 1 above)
Comment (bordia98) [reply js/main.js]: @pvgupta24 done in the next commit

**Pull request merged**